src/common/chat-participants/powerpages/PowerPagesChatParticipant.ts

```diff
@@ -257,5 +265,27 @@ export class PowerPagesChatParticipant {
         this.orgID = orgID;
         this.orgUrl = orgUrl;
         this.environmentID = environmentID;
+        
+        // Re-initialize endpoint information after org change
+        await this.initializeEndpoint();
+    }
+
+    private async initializeEndpoint(): Promise<boolean> {
+        try {
+            if (!this.orgID || !this.environmentID) {
+                return false;
+            }
+            
+            this.cachedEndpoint = await getEndpoint(this.orgID, this.environmentID, this.cachedEndpoint, this.powerPagesAgentSessionId);
```

Comment:
What's the use of `this.cachedEndpoint` being passed to `getEndpoint` function?

Reply:
No use, it was part of older implementation. Updated it.